2018-09-14 04:05:53.949 6698 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-14 04:05:53.952 6698 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-14 04:06:53.950 6698 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-14 04:06:53.954 6698 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.004s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-14 04:07:16.360 6698 DEBUG oslo_concurrency.lockutils [req-139fdba2-9ac9-46cc-abb6-a5d46de752e5 085d0877f3ff4269abd61f915150cf77 74e6a538b4564de48b41d0ae5e818844 - - -] Acquired semaphore "dhcp-agent-network-lock-7251fcba-dd5d-48b8-9750-d8214ba1383f" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-14 04:07:16.747 6698 DEBUG neutron.agent.dhcp.agent [req-dfabc7c7-b248-422c-92c4-3c75f228fdae - - - - -] Calling driver for network: 7251fcba-dd5d-48b8-9750-d8214ba1383f action: enable call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-14 04:07:16.749 6698 DEBUG neutron.agent.linux.utils [req-dfabc7c7-b248-422c-92c4-3c75f228fdae - - - - -] Unable to access /var/lib/neutron/dhcp/7251fcba-dd5d-48b8-9750-d8214ba1383f/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-14 04:07:16.750 6698 DEBUG neutron.agent.linux.dhcp [req-dfabc7c7-b248-422c-92c4-3c75f228fdae - - - - -] DHCP port dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-7251fcba-dd5d-48b8-9750-d8214ba1383f on network 7251fcba-dd5d-48b8-9750-d8214ba1383f does not yet exist. Checking for a reserved port. _setup_reserved_dhcp_port /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1272
2018-09-14 04:07:16.750 6698 DEBUG neutron.agent.linux.dhcp [req-dfabc7c7-b248-422c-92c4-3c75f228fdae - - - - -] DHCP port dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-7251fcba-dd5d-48b8-9750-d8214ba1383f on network 7251fcba-dd5d-48b8-9750-d8214ba1383f does not yet exist. Creating new one. _setup_new_dhcp_port /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1293
2018-09-14 04:07:19.978 6698 DEBUG neutron.agent.linux.utils [req-c834b026-7e89-4bc2-a969-71b3f05f6c88 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 04:07:19.996 6698 DEBUG neutron.agent.linux.utils [req-c834b026-7e89-4bc2-a969-71b3f05f6c88 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'add', 'qdhcp-7251fcba-dd5d-48b8-9750-d8214ba1383f'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 04:07:20.013 6698 DEBUG neutron.agent.linux.utils [req-c834b026-7e89-4bc2-a969-71b3f05f6c88 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7251fcba-dd5d-48b8-9750-d8214ba1383f', 'sysctl', '-w', 'net.ipv4.conf.all.promote_secondaries=1'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 04:07:20.028 6698 DEBUG neutron.agent.linux.utils [req-c834b026-7e89-4bc2-a969-71b3f05f6c88 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7251fcba-dd5d-48b8-9750-d8214ba1383f', 'ip', 'link', 'set', 'lo', 'up'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 04:07:20.042 6698 DEBUG neutron.agent.linux.utils [req-c834b026-7e89-4bc2-a969-71b3f05f6c88 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7251fcba-dd5d-48b8-9750-d8214ba1383f', 'sysctl', '-w', 'net.ipv6.conf.default.accept_ra=0'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 04:07:20.058 6698 DEBUG neutron.agent.linux.utils [req-c834b026-7e89-4bc2-a969-71b3f05f6c88 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7251fcba-dd5d-48b8-9750-d8214ba1383f', 'ip', 'link', 'set', 'tap1fd651e8-79', 'up'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 04:07:20.083 6698 DEBUG neutron.agent.linux.utils [req-c834b026-7e89-4bc2-a969-71b3f05f6c88 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7251fcba-dd5d-48b8-9750-d8214ba1383f', 'ip', '-o', 'link', 'show', 'tap1fd651e8-79'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 04:07:20.099 6698 DEBUG neutron.agent.linux.utils [req-c834b026-7e89-4bc2-a969-71b3f05f6c88 - - - - -] Running command: ['ip', '-o', 'link', 'show', 'br-int'] create_process /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:92
2018-09-14 04:07:20.120 6698 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=0): DelPortCommand(if_exists=True, bridge=None, port=tap1fd651e8-79) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-09-14 04:07:20.121 6698 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Transaction caused no change do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:114
2018-09-14 04:07:20.122 6698 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=0): AddPortCommand(bridge=br-int, may_exist=False, port=tap1fd651e8-79) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-09-14 04:07:20.123 6698 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=1): DbSetCommand(table=Port, col_values=(('tag', 4095),), record=tap1fd651e8-79) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-09-14 04:07:20.124 6698 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=2): DbSetCommand(table=Interface, col_values=(('type', 'internal'), ('external_ids', {'iface-id': u'1fd651e8-7935-4a2b-aa79-091b667da18d', 'iface-status': 'active', 'attached-mac': u'fa:16:3e:47:a4:f1'})), record=tap1fd651e8-79) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-09-14 04:07:20.148 6698 DEBUG neutron.agent.linux.utils [req-c834b026-7e89-4bc2-a969-71b3f05f6c88 - - - - -] Running command (rootwrap daemon): ['ip', 'link', 'set', 'tap1fd651e8-79', 'address', 'fa:16:3e:47:a4:f1'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 04:07:20.160 6698 DEBUG neutron.agent.linux.utils [req-c834b026-7e89-4bc2-a969-71b3f05f6c88 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 04:07:20.170 6698 DEBUG neutron.agent.linux.utils [req-c834b026-7e89-4bc2-a969-71b3f05f6c88 - - - - -] Running command (rootwrap daemon): ['ip', 'link', 'set', 'tap1fd651e8-79', 'netns', 'qdhcp-7251fcba-dd5d-48b8-9750-d8214ba1383f'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 04:07:20.189 6698 DEBUG neutron.agent.linux.utils [req-c834b026-7e89-4bc2-a969-71b3f05f6c88 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7251fcba-dd5d-48b8-9750-d8214ba1383f', 'ip', 'link', 'set', 'tap1fd651e8-79', 'mtu', '1450'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 04:07:20.200 6698 DEBUG neutron.agent.linux.utils [req-c834b026-7e89-4bc2-a969-71b3f05f6c88 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7251fcba-dd5d-48b8-9750-d8214ba1383f', 'ip', 'link', 'set', 'tap1fd651e8-79', 'up'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 04:07:20.213 6698 DEBUG oslo_concurrency.lockutils [req-c834b026-7e89-4bc2-a969-71b3f05f6c88 - - - - -] Acquired semaphore "iptables-qdhcp-7251fcba-dd5d-48b8-9750-d8214ba1383f" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-14 04:07:20.214 6698 DEBUG neutron.agent.linux.utils [req-c834b026-7e89-4bc2-a969-71b3f05f6c88 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7251fcba-dd5d-48b8-9750-d8214ba1383f', 'iptables-save'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 04:07:20.238 6698 DEBUG neutron.agent.linux.utils [req-c834b026-7e89-4bc2-a969-71b3f05f6c88 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7251fcba-dd5d-48b8-9750-d8214ba1383f', 'iptables-restore', '-n'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 04:07:20.256 6698 DEBUG neutron.agent.linux.iptables_manager [req-c834b026-7e89-4bc2-a969-71b3f05f6c88 - - - - -] IPTablesManager.apply completed with success. 56 iptables commands were issued _apply_synchronized /usr/lib/python2.7/site-packages/neutron/agent/linux/iptables_manager.py:601
2018-09-14 04:07:20.257 6698 DEBUG oslo_concurrency.lockutils [req-c834b026-7e89-4bc2-a969-71b3f05f6c88 - - - - -] Releasing semaphore "iptables-qdhcp-7251fcba-dd5d-48b8-9750-d8214ba1383f" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-14 04:07:20.258 6698 DEBUG neutron.agent.linux.utils [req-c834b026-7e89-4bc2-a969-71b3f05f6c88 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7251fcba-dd5d-48b8-9750-d8214ba1383f', 'ip', '-o', 'link', 'show', 'tap1fd651e8-79'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 04:07:20.270 6698 DEBUG neutron.agent.linux.utils [req-c834b026-7e89-4bc2-a969-71b3f05f6c88 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7251fcba-dd5d-48b8-9750-d8214ba1383f', 'ip', 'addr', 'show', 'tap1fd651e8-79'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 04:07:20.283 6698 DEBUG neutron.agent.linux.utils [req-c834b026-7e89-4bc2-a969-71b3f05f6c88 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7251fcba-dd5d-48b8-9750-d8214ba1383f', 'ip', '-4', 'addr', 'add', '11.197.1.2/24', 'scope', 'global', 'dev', 'tap1fd651e8-79', 'brd', '11.197.1.255'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 04:07:20.296 6698 DEBUG neutron.agent.linux.utils [req-c834b026-7e89-4bc2-a969-71b3f05f6c88 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7251fcba-dd5d-48b8-9750-d8214ba1383f', 'ip', '-4', 'route', 'list', 'dev', 'tap1fd651e8-79'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 04:07:20.308 6698 DEBUG neutron.agent.linux.dhcp [req-c834b026-7e89-4bc2-a969-71b3f05f6c88 - - - - -] Setting IPv4 gateway for dhcp netns on net 7251fcba-dd5d-48b8-9750-d8214ba1383f to 11.197.1.1 _set_default_route_ip_version /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1169
2018-09-14 04:07:20.309 6698 DEBUG neutron.agent.linux.utils [req-c834b026-7e89-4bc2-a969-71b3f05f6c88 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7251fcba-dd5d-48b8-9750-d8214ba1383f', 'ip', '-4', 'route', 'replace', 'default', 'via', '11.197.1.1', 'dev', 'tap1fd651e8-79'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 04:07:20.322 6698 DEBUG neutron.agent.linux.utils [req-c834b026-7e89-4bc2-a969-71b3f05f6c88 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7251fcba-dd5d-48b8-9750-d8214ba1383f', 'ip', '-6', 'route', 'list', 'dev', 'tap1fd651e8-79'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 04:07:20.334 6698 DEBUG neutron.agent.linux.dhcp [req-c834b026-7e89-4bc2-a969-71b3f05f6c88 - - - - -] Cleaning stale devices for network 7251fcba-dd5d-48b8-9750-d8214ba1383f _cleanup_stale_devices /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1374
2018-09-14 04:07:20.335 6698 DEBUG neutron.agent.linux.utils [req-c834b026-7e89-4bc2-a969-71b3f05f6c88 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 04:07:20.344 6698 DEBUG neutron.agent.linux.utils [req-c834b026-7e89-4bc2-a969-71b3f05f6c88 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7251fcba-dd5d-48b8-9750-d8214ba1383f', 'find', '/sys/class/net', '-maxdepth', '1', '-type', 'l', '-printf', '%f '] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 04:07:20.359 6698 DEBUG neutron.agent.linux.dhcp [req-c834b026-7e89-4bc2-a969-71b3f05f6c88 - - - - -] Building initial lease file: /var/lib/neutron/dhcp/7251fcba-dd5d-48b8-9750-d8214ba1383f/leases _output_init_lease_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:626
2018-09-14 04:07:20.361 6698 DEBUG neutron.agent.linux.dhcp [req-c834b026-7e89-4bc2-a969-71b3f05f6c88 - - - - -] Done building initial lease file /var/lib/neutron/dhcp/7251fcba-dd5d-48b8-9750-d8214ba1383f/leases with contents:
1536998840 fa:16:3e:47:a4:f1 11.197.1.2 * *
 _output_init_lease_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:654
2018-09-14 04:07:20.361 6698 DEBUG neutron.agent.linux.dhcp [req-c834b026-7e89-4bc2-a969-71b3f05f6c88 - - - - -] Building host file: /var/lib/neutron/dhcp/7251fcba-dd5d-48b8-9750-d8214ba1383f/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-14 04:07:20.363 6698 DEBUG neutron.agent.linux.dhcp [req-c834b026-7e89-4bc2-a969-71b3f05f6c88 - - - - -] Done building host file /var/lib/neutron/dhcp/7251fcba-dd5d-48b8-9750-d8214ba1383f/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-14 04:07:20.366 6698 DEBUG neutron.agent.linux.utils [req-c834b026-7e89-4bc2-a969-71b3f05f6c88 - - - - -] Unable to access /var/lib/neutron/dhcp/7251fcba-dd5d-48b8-9750-d8214ba1383f/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-14 04:07:20.368 6698 DEBUG neutron.agent.linux.utils [req-c834b026-7e89-4bc2-a969-71b3f05f6c88 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7251fcba-dd5d-48b8-9750-d8214ba1383f', 'dnsmasq', '--no-hosts', '--no-resolv', '--strict-order', '--except-interface=lo', '--pid-file=/var/lib/neutron/dhcp/7251fcba-dd5d-48b8-9750-d8214ba1383f/pid', '--dhcp-hostsfile=/var/lib/neutron/dhcp/7251fcba-dd5d-48b8-9750-d8214ba1383f/host', '--addn-hosts=/var/lib/neutron/dhcp/7251fcba-dd5d-48b8-9750-d8214ba1383f/addn_hosts', '--dhcp-optsfile=/var/lib/neutron/dhcp/7251fcba-dd5d-48b8-9750-d8214ba1383f/opts', '--dhcp-leasefile=/var/lib/neutron/dhcp/7251fcba-dd5d-48b8-9750-d8214ba1383f/leases', '--dhcp-match=set:ipxe,175', '--bind-interfaces', '--interface=tap1fd651e8-79', '--dhcp-range=set:tag0,11.197.1.0,static,255.255.255.0,86400s', '--dhcp-option-force=option:mtu,1450', '--dhcp-lease-max=256', '--conf-file=', '--domain=openstacklocal'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 04:07:20.400 6698 DEBUG neutron.agent.linux.utils [req-c834b026-7e89-4bc2-a969-71b3f05f6c88 - - - - -] Unable to access /var/lib/neutron/external/pids/7251fcba-dd5d-48b8-9750-d8214ba1383f.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-14 04:07:20.403 6698 DEBUG neutron.agent.linux.utils [req-c834b026-7e89-4bc2-a969-71b3f05f6c88 - - - - -] Unable to access /var/lib/neutron/external/pids/7251fcba-dd5d-48b8-9750-d8214ba1383f.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-14 04:07:20.404 6698 DEBUG neutron.agent.linux.external_process [req-c834b026-7e89-4bc2-a969-71b3f05f6c88 - - - - -] No process started for 7251fcba-dd5d-48b8-9750-d8214ba1383f disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-14 04:07:20.406 6698 DEBUG oslo_concurrency.lockutils [req-c834b026-7e89-4bc2-a969-71b3f05f6c88 - - - - -] Releasing semaphore "dhcp-agent-network-lock-7251fcba-dd5d-48b8-9750-d8214ba1383f" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-14 04:07:20.407 6698 DEBUG oslo_concurrency.lockutils [req-139fdba2-9ac9-46cc-abb6-a5d46de752e5 085d0877f3ff4269abd61f915150cf77 74e6a538b4564de48b41d0ae5e818844 - - -] Acquired semaphore "dhcp-agent-network-lock-7251fcba-dd5d-48b8-9750-d8214ba1383f" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-14 04:07:21.184 6698 DEBUG neutron.agent.dhcp.agent [req-9368f941-85c9-47f0-b924-421421e772b8 - - - - -] Calling driver for network: 7251fcba-dd5d-48b8-9750-d8214ba1383f action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-14 04:07:21.186 6698 DEBUG neutron.agent.linux.dhcp [req-9368f941-85c9-47f0-b924-421421e772b8 - - - - -] Building host file: /var/lib/neutron/dhcp/7251fcba-dd5d-48b8-9750-d8214ba1383f/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-14 04:07:21.190 6698 DEBUG neutron.agent.linux.dhcp [req-9368f941-85c9-47f0-b924-421421e772b8 - - - - -] Done building host file /var/lib/neutron/dhcp/7251fcba-dd5d-48b8-9750-d8214ba1383f/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-14 04:07:21.195 6698 DEBUG neutron.agent.linux.utils [req-9368f941-85c9-47f0-b924-421421e772b8 - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '15040'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 04:07:21.209 6698 DEBUG neutron.agent.linux.dhcp [req-9368f941-85c9-47f0-b924-421421e772b8 - - - - -] Reloading allocations for network: 7251fcba-dd5d-48b8-9750-d8214ba1383f reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-14 04:07:21.211 6698 DEBUG neutron.agent.linux.utils [req-9368f941-85c9-47f0-b924-421421e772b8 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7251fcba-dd5d-48b8-9750-d8214ba1383f', 'ip', '-4', 'route', 'list', 'dev', 'tap1fd651e8-79'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 04:07:21.226 6698 DEBUG neutron.agent.linux.utils [req-9368f941-85c9-47f0-b924-421421e772b8 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7251fcba-dd5d-48b8-9750-d8214ba1383f', 'ip', '-6', 'route', 'list', 'dev', 'tap1fd651e8-79'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 04:07:21.239 6698 DEBUG neutron.agent.linux.utils [req-9368f941-85c9-47f0-b924-421421e772b8 - - - - -] Unable to access /var/lib/neutron/external/pids/7251fcba-dd5d-48b8-9750-d8214ba1383f.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-14 04:07:21.240 6698 DEBUG neutron.agent.linux.utils [req-9368f941-85c9-47f0-b924-421421e772b8 - - - - -] Unable to access /var/lib/neutron/external/pids/7251fcba-dd5d-48b8-9750-d8214ba1383f.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-14 04:07:21.241 6698 DEBUG neutron.agent.linux.external_process [req-9368f941-85c9-47f0-b924-421421e772b8 - - - - -] No process started for 7251fcba-dd5d-48b8-9750-d8214ba1383f disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-14 04:07:21.242 6698 DEBUG oslo_concurrency.lockutils [req-9368f941-85c9-47f0-b924-421421e772b8 - - - - -] Releasing semaphore "dhcp-agent-network-lock-7251fcba-dd5d-48b8-9750-d8214ba1383f" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-14 04:07:21.243 6698 DEBUG oslo_concurrency.lockutils [req-c834b026-7e89-4bc2-a969-71b3f05f6c88 - - - - -] Acquired semaphore "dhcp-agent-network-lock-7251fcba-dd5d-48b8-9750-d8214ba1383f" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-14 04:07:21.244 6698 DEBUG neutron.agent.dhcp.agent [req-c834b026-7e89-4bc2-a969-71b3f05f6c88 - - - - -] Discarding stale port update: admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-09-14T08:07:17Z, description=, device_id=dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-7251fcba-dd5d-48b8-9750-d8214ba1383f, device_owner=network:dhcp, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'928c3ff7-8f5a-495c-ac7d-40341d2f7af2', u'ip_address': u'11.197.1.2'}], id=1fd651e8-7935-4a2b-aa79-091b667da18d, mac_address=fa:16:3e:47:a4:f1, name=, network_id=7251fcba-dd5d-48b8-9750-d8214ba1383f, port_security_enabled=False, project_id=74e6a538b4564de48b41d0ae5e818844, revision_number=3, security_groups=[], status=DOWN, tags=[], tenant_id=74e6a538b4564de48b41d0ae5e818844, updated_at=2018-09-14T08:07:18Z port_update_end /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:410
2018-09-14 04:07:21.244 6698 DEBUG oslo_concurrency.lockutils [req-c834b026-7e89-4bc2-a969-71b3f05f6c88 - - - - -] Releasing semaphore "dhcp-agent-network-lock-7251fcba-dd5d-48b8-9750-d8214ba1383f" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-14 04:07:21.245 6698 DEBUG oslo_concurrency.lockutils [req-c834b026-7e89-4bc2-a969-71b3f05f6c88 - - - - -] Acquired semaphore "dhcp-agent-network-lock-7251fcba-dd5d-48b8-9750-d8214ba1383f" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-14 04:07:21.246 6698 INFO neutron.agent.dhcp.agent [req-c834b026-7e89-4bc2-a969-71b3f05f6c88 - - - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=datapath_type=system, ovs_hybrid_plug=True, port_filter=True, binding:vif_type=ovs, binding:vnic_type=normal, created_at=2018-09-14T08:07:17Z, description=, device_id=dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-7251fcba-dd5d-48b8-9750-d8214ba1383f, device_owner=network:dhcp, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'928c3ff7-8f5a-495c-ac7d-40341d2f7af2', u'ip_address': u'11.197.1.2'}], id=1fd651e8-7935-4a2b-aa79-091b667da18d, mac_address=fa:16:3e:47:a4:f1, name=, network_id=7251fcba-dd5d-48b8-9750-d8214ba1383f, port_security_enabled=False, project_id=74e6a538b4564de48b41d0ae5e818844, revision_number=4, security_groups=[], status=DOWN, tags=[], tenant_id=74e6a538b4564de48b41d0ae5e818844, updated_at=2018-09-14T08:07:19Z
2018-09-14 04:07:21.248 6698 DEBUG neutron.agent.dhcp.agent [req-c834b026-7e89-4bc2-a969-71b3f05f6c88 - - - - -] Calling driver for network: 7251fcba-dd5d-48b8-9750-d8214ba1383f action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-14 04:07:21.249 6698 DEBUG neutron.agent.linux.dhcp [req-c834b026-7e89-4bc2-a969-71b3f05f6c88 - - - - -] Building host file: /var/lib/neutron/dhcp/7251fcba-dd5d-48b8-9750-d8214ba1383f/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-14 04:07:21.251 6698 DEBUG neutron.agent.linux.dhcp [req-c834b026-7e89-4bc2-a969-71b3f05f6c88 - - - - -] Done building host file /var/lib/neutron/dhcp/7251fcba-dd5d-48b8-9750-d8214ba1383f/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-14 04:07:21.254 6698 DEBUG neutron.agent.linux.utils [req-c834b026-7e89-4bc2-a969-71b3f05f6c88 - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '15040'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 04:07:21.267 6698 DEBUG neutron.agent.linux.dhcp [req-c834b026-7e89-4bc2-a969-71b3f05f6c88 - - - - -] Reloading allocations for network: 7251fcba-dd5d-48b8-9750-d8214ba1383f reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-14 04:07:21.268 6698 DEBUG neutron.agent.linux.utils [req-c834b026-7e89-4bc2-a969-71b3f05f6c88 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7251fcba-dd5d-48b8-9750-d8214ba1383f', 'ip', '-4', 'route', 'list', 'dev', 'tap1fd651e8-79'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 04:07:21.279 6698 DEBUG neutron.agent.linux.utils [req-c834b026-7e89-4bc2-a969-71b3f05f6c88 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7251fcba-dd5d-48b8-9750-d8214ba1383f', 'ip', '-6', 'route', 'list', 'dev', 'tap1fd651e8-79'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 04:07:21.291 6698 DEBUG neutron.agent.linux.utils [req-c834b026-7e89-4bc2-a969-71b3f05f6c88 - - - - -] Unable to access /var/lib/neutron/external/pids/7251fcba-dd5d-48b8-9750-d8214ba1383f.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-14 04:07:21.292 6698 DEBUG neutron.agent.linux.utils [req-c834b026-7e89-4bc2-a969-71b3f05f6c88 - - - - -] Unable to access /var/lib/neutron/external/pids/7251fcba-dd5d-48b8-9750-d8214ba1383f.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-14 04:07:21.293 6698 DEBUG neutron.agent.linux.external_process [req-c834b026-7e89-4bc2-a969-71b3f05f6c88 - - - - -] No process started for 7251fcba-dd5d-48b8-9750-d8214ba1383f disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-14 04:07:21.293 6698 DEBUG oslo_concurrency.lockutils [req-c834b026-7e89-4bc2-a969-71b3f05f6c88 - - - - -] Releasing semaphore "dhcp-agent-network-lock-7251fcba-dd5d-48b8-9750-d8214ba1383f" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-14 04:07:53.675 6698 DEBUG oslo_concurrency.lockutils [req-0b200922-f382-4771-81a4-be14ee03b0e4 085d0877f3ff4269abd61f915150cf77 74e6a538b4564de48b41d0ae5e818844 - - -] Acquired semaphore "dhcp-agent-network-lock-7251fcba-dd5d-48b8-9750-d8214ba1383f" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-14 04:07:53.676 6698 INFO neutron.agent.dhcp.agent [req-0b200922-f382-4771-81a4-be14ee03b0e4 085d0877f3ff4269abd61f915150cf77 74e6a538b4564de48b41d0ae5e818844 - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-09-14T08:07:52Z, description=, device_id=5255b721-caf8-4e39-8891-7a3001d12972, device_owner=network:router_interface, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'928c3ff7-8f5a-495c-ac7d-40341d2f7af2', u'ip_address': u'11.197.1.1'}], id=f932038c-d006-4608-ba66-da5aa8ad90ae, mac_address=fa:16:3e:f8:ff:59, name=, network_id=7251fcba-dd5d-48b8-9750-d8214ba1383f, port_security_enabled=False, project_id=74e6a538b4564de48b41d0ae5e818844, revision_number=3, security_groups=[], status=DOWN, tags=[], tenant_id=74e6a538b4564de48b41d0ae5e818844, updated_at=2018-09-14T08:07:53Z
2018-09-14 04:07:53.679 6698 DEBUG neutron.agent.dhcp.agent [req-0b200922-f382-4771-81a4-be14ee03b0e4 085d0877f3ff4269abd61f915150cf77 74e6a538b4564de48b41d0ae5e818844 - - -] Calling driver for network: 7251fcba-dd5d-48b8-9750-d8214ba1383f action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-14 04:07:53.681 6698 DEBUG neutron.agent.linux.dhcp [req-0b200922-f382-4771-81a4-be14ee03b0e4 085d0877f3ff4269abd61f915150cf77 74e6a538b4564de48b41d0ae5e818844 - - -] Building host file: /var/lib/neutron/dhcp/7251fcba-dd5d-48b8-9750-d8214ba1383f/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-14 04:07:53.685 6698 DEBUG neutron.agent.linux.dhcp [req-0b200922-f382-4771-81a4-be14ee03b0e4 085d0877f3ff4269abd61f915150cf77 74e6a538b4564de48b41d0ae5e818844 - - -] Done building host file /var/lib/neutron/dhcp/7251fcba-dd5d-48b8-9750-d8214ba1383f/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-14 04:07:53.691 6698 DEBUG neutron.agent.linux.utils [req-0b200922-f382-4771-81a4-be14ee03b0e4 085d0877f3ff4269abd61f915150cf77 74e6a538b4564de48b41d0ae5e818844 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '15040'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 04:07:53.728 6698 DEBUG neutron.agent.linux.dhcp [req-0b200922-f382-4771-81a4-be14ee03b0e4 085d0877f3ff4269abd61f915150cf77 74e6a538b4564de48b41d0ae5e818844 - - -] Reloading allocations for network: 7251fcba-dd5d-48b8-9750-d8214ba1383f reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-14 04:07:53.733 6698 DEBUG neutron.agent.linux.utils [req-0b200922-f382-4771-81a4-be14ee03b0e4 085d0877f3ff4269abd61f915150cf77 74e6a538b4564de48b41d0ae5e818844 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7251fcba-dd5d-48b8-9750-d8214ba1383f', 'ip', '-4', 'route', 'list', 'dev', 'tap1fd651e8-79'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 04:07:53.758 6698 DEBUG neutron.agent.linux.utils [req-0b200922-f382-4771-81a4-be14ee03b0e4 085d0877f3ff4269abd61f915150cf77 74e6a538b4564de48b41d0ae5e818844 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7251fcba-dd5d-48b8-9750-d8214ba1383f', 'ip', '-6', 'route', 'list', 'dev', 'tap1fd651e8-79'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 04:07:53.776 6698 DEBUG neutron.agent.linux.utils [req-0b200922-f382-4771-81a4-be14ee03b0e4 085d0877f3ff4269abd61f915150cf77 74e6a538b4564de48b41d0ae5e818844 - - -] Unable to access /var/lib/neutron/external/pids/7251fcba-dd5d-48b8-9750-d8214ba1383f.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-14 04:07:53.778 6698 DEBUG neutron.agent.linux.utils [req-0b200922-f382-4771-81a4-be14ee03b0e4 085d0877f3ff4269abd61f915150cf77 74e6a538b4564de48b41d0ae5e818844 - - -] Unable to access /var/lib/neutron/external/pids/7251fcba-dd5d-48b8-9750-d8214ba1383f.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-14 04:07:53.780 6698 DEBUG neutron.agent.linux.external_process [req-0b200922-f382-4771-81a4-be14ee03b0e4 085d0877f3ff4269abd61f915150cf77 74e6a538b4564de48b41d0ae5e818844 - - -] No process started for 7251fcba-dd5d-48b8-9750-d8214ba1383f disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-14 04:07:53.781 6698 DEBUG oslo_concurrency.lockutils [req-0b200922-f382-4771-81a4-be14ee03b0e4 085d0877f3ff4269abd61f915150cf77 74e6a538b4564de48b41d0ae5e818844 - - -] Releasing semaphore "dhcp-agent-network-lock-7251fcba-dd5d-48b8-9750-d8214ba1383f" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-14 04:07:53.951 6698 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.002s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-14 04:07:53.957 6698 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.006s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-14 04:07:56.013 6698 DEBUG oslo_concurrency.lockutils [req-0b200922-f382-4771-81a4-be14ee03b0e4 085d0877f3ff4269abd61f915150cf77 74e6a538b4564de48b41d0ae5e818844 - - -] Acquired semaphore "dhcp-agent-network-lock-7251fcba-dd5d-48b8-9750-d8214ba1383f" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-14 04:07:56.014 6698 INFO neutron.agent.dhcp.agent [req-0b200922-f382-4771-81a4-be14ee03b0e4 085d0877f3ff4269abd61f915150cf77 74e6a538b4564de48b41d0ae5e818844 - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-09-14T08:07:52Z, description=, device_id=5255b721-caf8-4e39-8891-7a3001d12972, device_owner=network:router_interface, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'928c3ff7-8f5a-495c-ac7d-40341d2f7af2', u'ip_address': u'11.197.1.1'}], id=f932038c-d006-4608-ba66-da5aa8ad90ae, mac_address=fa:16:3e:f8:ff:59, name=, network_id=7251fcba-dd5d-48b8-9750-d8214ba1383f, port_security_enabled=False, project_id=74e6a538b4564de48b41d0ae5e818844, revision_number=3, security_groups=[], status=DOWN, tags=[], tenant_id=74e6a538b4564de48b41d0ae5e818844, updated_at=2018-09-14T08:07:53Z
2018-09-14 04:07:56.016 6698 DEBUG neutron.agent.dhcp.agent [req-0b200922-f382-4771-81a4-be14ee03b0e4 085d0877f3ff4269abd61f915150cf77 74e6a538b4564de48b41d0ae5e818844 - - -] Calling driver for network: 7251fcba-dd5d-48b8-9750-d8214ba1383f action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-14 04:07:56.018 6698 DEBUG neutron.agent.linux.dhcp [req-0b200922-f382-4771-81a4-be14ee03b0e4 085d0877f3ff4269abd61f915150cf77 74e6a538b4564de48b41d0ae5e818844 - - -] Building host file: /var/lib/neutron/dhcp/7251fcba-dd5d-48b8-9750-d8214ba1383f/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-14 04:07:56.021 6698 DEBUG neutron.agent.linux.dhcp [req-0b200922-f382-4771-81a4-be14ee03b0e4 085d0877f3ff4269abd61f915150cf77 74e6a538b4564de48b41d0ae5e818844 - - -] Done building host file /var/lib/neutron/dhcp/7251fcba-dd5d-48b8-9750-d8214ba1383f/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-14 04:07:56.029 6698 DEBUG neutron.agent.linux.utils [req-0b200922-f382-4771-81a4-be14ee03b0e4 085d0877f3ff4269abd61f915150cf77 74e6a538b4564de48b41d0ae5e818844 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '15040'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 04:07:56.045 6698 DEBUG neutron.agent.linux.dhcp [req-0b200922-f382-4771-81a4-be14ee03b0e4 085d0877f3ff4269abd61f915150cf77 74e6a538b4564de48b41d0ae5e818844 - - -] Reloading allocations for network: 7251fcba-dd5d-48b8-9750-d8214ba1383f reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-14 04:07:56.046 6698 DEBUG neutron.agent.linux.utils [req-0b200922-f382-4771-81a4-be14ee03b0e4 085d0877f3ff4269abd61f915150cf77 74e6a538b4564de48b41d0ae5e818844 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7251fcba-dd5d-48b8-9750-d8214ba1383f', 'ip', '-4', 'route', 'list', 'dev', 'tap1fd651e8-79'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 04:07:56.071 6698 DEBUG neutron.agent.linux.utils [req-0b200922-f382-4771-81a4-be14ee03b0e4 085d0877f3ff4269abd61f915150cf77 74e6a538b4564de48b41d0ae5e818844 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7251fcba-dd5d-48b8-9750-d8214ba1383f', 'ip', '-6', 'route', 'list', 'dev', 'tap1fd651e8-79'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 04:07:56.092 6698 DEBUG neutron.agent.linux.utils [req-0b200922-f382-4771-81a4-be14ee03b0e4 085d0877f3ff4269abd61f915150cf77 74e6a538b4564de48b41d0ae5e818844 - - -] Unable to access /var/lib/neutron/external/pids/7251fcba-dd5d-48b8-9750-d8214ba1383f.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-14 04:07:56.093 6698 DEBUG neutron.agent.linux.utils [req-0b200922-f382-4771-81a4-be14ee03b0e4 085d0877f3ff4269abd61f915150cf77 74e6a538b4564de48b41d0ae5e818844 - - -] Unable to access /var/lib/neutron/external/pids/7251fcba-dd5d-48b8-9750-d8214ba1383f.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-14 04:07:56.094 6698 DEBUG neutron.agent.linux.external_process [req-0b200922-f382-4771-81a4-be14ee03b0e4 085d0877f3ff4269abd61f915150cf77 74e6a538b4564de48b41d0ae5e818844 - - -] No process started for 7251fcba-dd5d-48b8-9750-d8214ba1383f disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-14 04:07:56.096 6698 DEBUG oslo_concurrency.lockutils [req-0b200922-f382-4771-81a4-be14ee03b0e4 085d0877f3ff4269abd61f915150cf77 74e6a538b4564de48b41d0ae5e818844 - - -] Releasing semaphore "dhcp-agent-network-lock-7251fcba-dd5d-48b8-9750-d8214ba1383f" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-14 04:07:59.648 6698 DEBUG oslo_concurrency.lockutils [req-252ab348-9409-4b60-ba83-0de9baf194d2 - - - - -] Acquired semaphore "dhcp-agent-network-lock-7251fcba-dd5d-48b8-9750-d8214ba1383f" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-14 04:07:59.649 6698 INFO neutron.agent.dhcp.agent [req-252ab348-9409-4b60-ba83-0de9baf194d2 - - - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-09-14T08:07:52Z, description=, device_id=5255b721-caf8-4e39-8891-7a3001d12972, device_owner=network:router_interface, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'928c3ff7-8f5a-495c-ac7d-40341d2f7af2', u'ip_address': u'11.197.1.1'}], id=f932038c-d006-4608-ba66-da5aa8ad90ae, mac_address=fa:16:3e:f8:ff:59, name=, network_id=7251fcba-dd5d-48b8-9750-d8214ba1383f, port_security_enabled=False, project_id=74e6a538b4564de48b41d0ae5e818844, revision_number=4, security_groups=[], status=DOWN, tags=[], tenant_id=74e6a538b4564de48b41d0ae5e818844, updated_at=2018-09-14T08:07:54Z
2018-09-14 04:07:59.650 6698 DEBUG neutron.agent.dhcp.agent [req-252ab348-9409-4b60-ba83-0de9baf194d2 - - - - -] Calling driver for network: 7251fcba-dd5d-48b8-9750-d8214ba1383f action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-14 04:07:59.652 6698 DEBUG neutron.agent.linux.dhcp [req-252ab348-9409-4b60-ba83-0de9baf194d2 - - - - -] Building host file: /var/lib/neutron/dhcp/7251fcba-dd5d-48b8-9750-d8214ba1383f/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-14 04:07:59.653 6698 DEBUG neutron.agent.linux.dhcp [req-252ab348-9409-4b60-ba83-0de9baf194d2 - - - - -] Done building host file /var/lib/neutron/dhcp/7251fcba-dd5d-48b8-9750-d8214ba1383f/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-14 04:07:59.657 6698 DEBUG neutron.agent.linux.utils [req-252ab348-9409-4b60-ba83-0de9baf194d2 - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '15040'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 04:07:59.674 6698 DEBUG neutron.agent.linux.dhcp [req-252ab348-9409-4b60-ba83-0de9baf194d2 - - - - -] Reloading allocations for network: 7251fcba-dd5d-48b8-9750-d8214ba1383f reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-14 04:07:59.675 6698 DEBUG neutron.agent.linux.utils [req-252ab348-9409-4b60-ba83-0de9baf194d2 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7251fcba-dd5d-48b8-9750-d8214ba1383f', 'ip', '-4', 'route', 'list', 'dev', 'tap1fd651e8-79'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 04:07:59.694 6698 DEBUG neutron.agent.linux.utils [req-252ab348-9409-4b60-ba83-0de9baf194d2 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7251fcba-dd5d-48b8-9750-d8214ba1383f', 'ip', '-6', 'route', 'list', 'dev', 'tap1fd651e8-79'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 04:07:59.711 6698 DEBUG neutron.agent.linux.utils [req-252ab348-9409-4b60-ba83-0de9baf194d2 - - - - -] Unable to access /var/lib/neutron/external/pids/7251fcba-dd5d-48b8-9750-d8214ba1383f.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-14 04:07:59.712 6698 DEBUG neutron.agent.linux.utils [req-252ab348-9409-4b60-ba83-0de9baf194d2 - - - - -] Unable to access /var/lib/neutron/external/pids/7251fcba-dd5d-48b8-9750-d8214ba1383f.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-14 04:07:59.713 6698 DEBUG neutron.agent.linux.external_process [req-252ab348-9409-4b60-ba83-0de9baf194d2 - - - - -] No process started for 7251fcba-dd5d-48b8-9750-d8214ba1383f disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-14 04:07:59.713 6698 DEBUG oslo_concurrency.lockutils [req-252ab348-9409-4b60-ba83-0de9baf194d2 - - - - -] Releasing semaphore "dhcp-agent-network-lock-7251fcba-dd5d-48b8-9750-d8214ba1383f" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-14 04:08:00.437 6698 DEBUG oslo_concurrency.lockutils [req-252ab348-9409-4b60-ba83-0de9baf194d2 - - - - -] Acquired semaphore "dhcp-agent-network-lock-7251fcba-dd5d-48b8-9750-d8214ba1383f" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-14 04:08:00.439 6698 INFO neutron.agent.dhcp.agent [req-252ab348-9409-4b60-ba83-0de9baf194d2 - - - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=datapath_type=system, ovs_hybrid_plug=True, port_filter=True, binding:vif_type=ovs, binding:vnic_type=normal, created_at=2018-09-14T08:07:52Z, description=, device_id=5255b721-caf8-4e39-8891-7a3001d12972, device_owner=network:router_interface, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'928c3ff7-8f5a-495c-ac7d-40341d2f7af2', u'ip_address': u'11.197.1.1'}], id=f932038c-d006-4608-ba66-da5aa8ad90ae, mac_address=fa:16:3e:f8:ff:59, name=, network_id=7251fcba-dd5d-48b8-9750-d8214ba1383f, port_security_enabled=False, project_id=74e6a538b4564de48b41d0ae5e818844, revision_number=6, security_groups=[], status=DOWN, tags=[], tenant_id=74e6a538b4564de48b41d0ae5e818844, updated_at=2018-09-14T08:08:00Z
2018-09-14 04:08:00.441 6698 DEBUG neutron.agent.dhcp.agent [req-252ab348-9409-4b60-ba83-0de9baf194d2 - - - - -] Calling driver for network: 7251fcba-dd5d-48b8-9750-d8214ba1383f action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-14 04:08:00.442 6698 DEBUG neutron.agent.linux.dhcp [req-252ab348-9409-4b60-ba83-0de9baf194d2 - - - - -] Building host file: /var/lib/neutron/dhcp/7251fcba-dd5d-48b8-9750-d8214ba1383f/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-14 04:08:00.445 6698 DEBUG neutron.agent.linux.dhcp [req-252ab348-9409-4b60-ba83-0de9baf194d2 - - - - -] Done building host file /var/lib/neutron/dhcp/7251fcba-dd5d-48b8-9750-d8214ba1383f/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-14 04:08:00.450 6698 DEBUG neutron.agent.linux.utils [req-252ab348-9409-4b60-ba83-0de9baf194d2 - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '15040'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 04:08:00.464 6698 DEBUG neutron.agent.linux.dhcp [req-252ab348-9409-4b60-ba83-0de9baf194d2 - - - - -] Reloading allocations for network: 7251fcba-dd5d-48b8-9750-d8214ba1383f reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-14 04:08:00.466 6698 DEBUG neutron.agent.linux.utils [req-252ab348-9409-4b60-ba83-0de9baf194d2 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7251fcba-dd5d-48b8-9750-d8214ba1383f', 'ip', '-4', 'route', 'list', 'dev', 'tap1fd651e8-79'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 04:08:00.482 6698 DEBUG neutron.agent.linux.utils [req-252ab348-9409-4b60-ba83-0de9baf194d2 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7251fcba-dd5d-48b8-9750-d8214ba1383f', 'ip', '-6', 'route', 'list', 'dev', 'tap1fd651e8-79'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 04:08:00.498 6698 DEBUG neutron.agent.linux.utils [req-252ab348-9409-4b60-ba83-0de9baf194d2 - - - - -] Unable to access /var/lib/neutron/external/pids/7251fcba-dd5d-48b8-9750-d8214ba1383f.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-14 04:08:00.499 6698 DEBUG neutron.agent.linux.utils [req-252ab348-9409-4b60-ba83-0de9baf194d2 - - - - -] Unable to access /var/lib/neutron/external/pids/7251fcba-dd5d-48b8-9750-d8214ba1383f.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-14 04:08:00.500 6698 DEBUG neutron.agent.linux.external_process [req-252ab348-9409-4b60-ba83-0de9baf194d2 - - - - -] No process started for 7251fcba-dd5d-48b8-9750-d8214ba1383f disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-14 04:08:00.501 6698 DEBUG oslo_concurrency.lockutils [req-252ab348-9409-4b60-ba83-0de9baf194d2 - - - - -] Releasing semaphore "dhcp-agent-network-lock-7251fcba-dd5d-48b8-9750-d8214ba1383f" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-14 04:08:06.840 6698 DEBUG oslo_concurrency.lockutils [req-e55836f7-b499-442a-ba27-2a56f1f66de5 085d0877f3ff4269abd61f915150cf77 74e6a538b4564de48b41d0ae5e818844 - - -] Acquired semaphore "dhcp-agent-network-lock-990b4ddf-dd04-44e0-920c-02fce451b987" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-14 04:08:06.841 6698 DEBUG oslo_concurrency.lockutils [req-e55836f7-b499-442a-ba27-2a56f1f66de5 085d0877f3ff4269abd61f915150cf77 74e6a538b4564de48b41d0ae5e818844 - - -] Releasing semaphore "dhcp-agent-network-lock-990b4ddf-dd04-44e0-920c-02fce451b987" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-14 04:08:09.195 6698 DEBUG oslo_concurrency.lockutils [req-2eb50d3d-e0d1-455d-8ff9-fa56bb6850f0 - - - - -] Acquired semaphore "dhcp-agent-network-lock-990b4ddf-dd04-44e0-920c-02fce451b987" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-14 04:08:09.196 6698 DEBUG oslo_concurrency.lockutils [req-2eb50d3d-e0d1-455d-8ff9-fa56bb6850f0 - - - - -] Releasing semaphore "dhcp-agent-network-lock-990b4ddf-dd04-44e0-920c-02fce451b987" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-14 04:08:09.942 6698 DEBUG oslo_concurrency.lockutils [req-2eb50d3d-e0d1-455d-8ff9-fa56bb6850f0 - - - - -] Acquired semaphore "dhcp-agent-network-lock-990b4ddf-dd04-44e0-920c-02fce451b987" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-14 04:08:09.943 6698 DEBUG oslo_concurrency.lockutils [req-2eb50d3d-e0d1-455d-8ff9-fa56bb6850f0 - - - - -] Releasing semaphore "dhcp-agent-network-lock-990b4ddf-dd04-44e0-920c-02fce451b987" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-14 04:08:26.451 6698 DEBUG oslo_concurrency.lockutils [req-738bd5d1-44cf-4685-96d7-caeac8247596 085d0877f3ff4269abd61f915150cf77 74e6a538b4564de48b41d0ae5e818844 - - -] Acquired semaphore "dhcp-agent-network-lock-7251fcba-dd5d-48b8-9750-d8214ba1383f" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-14 04:08:26.452 6698 INFO neutron.agent.dhcp.agent [req-738bd5d1-44cf-4685-96d7-caeac8247596 085d0877f3ff4269abd61f915150cf77 74e6a538b4564de48b41d0ae5e818844 - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-09-14T08:08:25Z, description=, device_id=f9001cef-311c-48d8-8014-c159f6526655, device_owner=, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'928c3ff7-8f5a-495c-ac7d-40341d2f7af2', u'ip_address': u'11.197.1.12'}], id=680456d8-5d4f-436f-a825-584188fd39ff, mac_address=fa:16:3e:0d:b7:f1, name=, network_id=7251fcba-dd5d-48b8-9750-d8214ba1383f, port_security_enabled=True, project_id=74e6a538b4564de48b41d0ae5e818844, revision_number=3, security_groups=[u'f45c25cf-f7a2-4b93-8eae-e467548ee8a4'], status=DOWN, tags=[], tenant_id=74e6a538b4564de48b41d0ae5e818844, updated_at=2018-09-14T08:08:25Z
2018-09-14 04:08:26.453 6698 DEBUG neutron.agent.dhcp.agent [req-738bd5d1-44cf-4685-96d7-caeac8247596 085d0877f3ff4269abd61f915150cf77 74e6a538b4564de48b41d0ae5e818844 - - -] Calling driver for network: 7251fcba-dd5d-48b8-9750-d8214ba1383f action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-14 04:08:26.455 6698 DEBUG neutron.agent.linux.dhcp [req-738bd5d1-44cf-4685-96d7-caeac8247596 085d0877f3ff4269abd61f915150cf77 74e6a538b4564de48b41d0ae5e818844 - - -] Building host file: /var/lib/neutron/dhcp/7251fcba-dd5d-48b8-9750-d8214ba1383f/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-14 04:08:26.459 6698 DEBUG neutron.agent.linux.dhcp [req-738bd5d1-44cf-4685-96d7-caeac8247596 085d0877f3ff4269abd61f915150cf77 74e6a538b4564de48b41d0ae5e818844 - - -] Done building host file /var/lib/neutron/dhcp/7251fcba-dd5d-48b8-9750-d8214ba1383f/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-14 04:08:26.464 6698 DEBUG neutron.agent.linux.utils [req-738bd5d1-44cf-4685-96d7-caeac8247596 085d0877f3ff4269abd61f915150cf77 74e6a538b4564de48b41d0ae5e818844 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '15040'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 04:08:26.486 6698 DEBUG neutron.agent.linux.dhcp [req-738bd5d1-44cf-4685-96d7-caeac8247596 085d0877f3ff4269abd61f915150cf77 74e6a538b4564de48b41d0ae5e818844 - - -] Reloading allocations for network: 7251fcba-dd5d-48b8-9750-d8214ba1383f reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-14 04:08:26.489 6698 DEBUG neutron.agent.linux.utils [req-738bd5d1-44cf-4685-96d7-caeac8247596 085d0877f3ff4269abd61f915150cf77 74e6a538b4564de48b41d0ae5e818844 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7251fcba-dd5d-48b8-9750-d8214ba1383f', 'ip', '-4', 'route', 'list', 'dev', 'tap1fd651e8-79'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 04:08:26.510 6698 DEBUG neutron.agent.linux.utils [req-738bd5d1-44cf-4685-96d7-caeac8247596 085d0877f3ff4269abd61f915150cf77 74e6a538b4564de48b41d0ae5e818844 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7251fcba-dd5d-48b8-9750-d8214ba1383f', 'ip', '-6', 'route', 'list', 'dev', 'tap1fd651e8-79'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 04:08:26.524 6698 DEBUG neutron.agent.linux.utils [req-738bd5d1-44cf-4685-96d7-caeac8247596 085d0877f3ff4269abd61f915150cf77 74e6a538b4564de48b41d0ae5e818844 - - -] Unable to access /var/lib/neutron/external/pids/7251fcba-dd5d-48b8-9750-d8214ba1383f.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-14 04:08:26.525 6698 DEBUG neutron.agent.linux.utils [req-738bd5d1-44cf-4685-96d7-caeac8247596 085d0877f3ff4269abd61f915150cf77 74e6a538b4564de48b41d0ae5e818844 - - -] Unable to access /var/lib/neutron/external/pids/7251fcba-dd5d-48b8-9750-d8214ba1383f.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-14 04:08:26.526 6698 DEBUG neutron.agent.linux.external_process [req-738bd5d1-44cf-4685-96d7-caeac8247596 085d0877f3ff4269abd61f915150cf77 74e6a538b4564de48b41d0ae5e818844 - - -] No process started for 7251fcba-dd5d-48b8-9750-d8214ba1383f disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-14 04:08:26.526 6698 DEBUG oslo_concurrency.lockutils [req-738bd5d1-44cf-4685-96d7-caeac8247596 085d0877f3ff4269abd61f915150cf77 74e6a538b4564de48b41d0ae5e818844 - - -] Releasing semaphore "dhcp-agent-network-lock-7251fcba-dd5d-48b8-9750-d8214ba1383f" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-14 04:08:28.462 6698 DEBUG oslo_concurrency.lockutils [req-673313b6-6aed-429e-8a6f-cb98c7430e09 10c05a22cea847f7a42d788fe8f1cd2e 368f193974e14846a186e2cbdb34d110 - - -] Acquired semaphore "dhcp-agent-network-lock-7251fcba-dd5d-48b8-9750-d8214ba1383f" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-14 04:08:28.462 6698 INFO neutron.agent.dhcp.agent [req-673313b6-6aed-429e-8a6f-cb98c7430e09 10c05a22cea847f7a42d788fe8f1cd2e 368f193974e14846a186e2cbdb34d110 - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-09-14T08:08:25Z, description=, device_id=f9001cef-311c-48d8-8014-c159f6526655, device_owner=compute:nova, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'928c3ff7-8f5a-495c-ac7d-40341d2f7af2', u'ip_address': u'11.197.1.12'}], id=680456d8-5d4f-436f-a825-584188fd39ff, mac_address=fa:16:3e:0d:b7:f1, name=, network_id=7251fcba-dd5d-48b8-9750-d8214ba1383f, port_security_enabled=True, project_id=74e6a538b4564de48b41d0ae5e818844, revision_number=4, security_groups=[u'f45c25cf-f7a2-4b93-8eae-e467548ee8a4'], status=DOWN, tags=[], tenant_id=74e6a538b4564de48b41d0ae5e818844, updated_at=2018-09-14T08:08:27Z
2018-09-14 04:08:28.463 6698 DEBUG neutron.agent.dhcp.agent [req-673313b6-6aed-429e-8a6f-cb98c7430e09 10c05a22cea847f7a42d788fe8f1cd2e 368f193974e14846a186e2cbdb34d110 - - -] Calling driver for network: 7251fcba-dd5d-48b8-9750-d8214ba1383f action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-14 04:08:28.465 6698 DEBUG neutron.agent.linux.dhcp [req-673313b6-6aed-429e-8a6f-cb98c7430e09 10c05a22cea847f7a42d788fe8f1cd2e 368f193974e14846a186e2cbdb34d110 - - -] Building host file: /var/lib/neutron/dhcp/7251fcba-dd5d-48b8-9750-d8214ba1383f/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-14 04:08:28.467 6698 DEBUG neutron.agent.linux.dhcp [req-673313b6-6aed-429e-8a6f-cb98c7430e09 10c05a22cea847f7a42d788fe8f1cd2e 368f193974e14846a186e2cbdb34d110 - - -] Done building host file /var/lib/neutron/dhcp/7251fcba-dd5d-48b8-9750-d8214ba1383f/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-14 04:08:28.471 6698 DEBUG neutron.agent.linux.utils [req-673313b6-6aed-429e-8a6f-cb98c7430e09 10c05a22cea847f7a42d788fe8f1cd2e 368f193974e14846a186e2cbdb34d110 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '15040'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 04:08:28.485 6698 DEBUG neutron.agent.linux.dhcp [req-673313b6-6aed-429e-8a6f-cb98c7430e09 10c05a22cea847f7a42d788fe8f1cd2e 368f193974e14846a186e2cbdb34d110 - - -] Reloading allocations for network: 7251fcba-dd5d-48b8-9750-d8214ba1383f reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-14 04:08:28.486 6698 DEBUG neutron.agent.linux.utils [req-673313b6-6aed-429e-8a6f-cb98c7430e09 10c05a22cea847f7a42d788fe8f1cd2e 368f193974e14846a186e2cbdb34d110 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7251fcba-dd5d-48b8-9750-d8214ba1383f', 'ip', '-4', 'route', 'list', 'dev', 'tap1fd651e8-79'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 04:08:28.498 6698 DEBUG neutron.agent.linux.utils [req-673313b6-6aed-429e-8a6f-cb98c7430e09 10c05a22cea847f7a42d788fe8f1cd2e 368f193974e14846a186e2cbdb34d110 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7251fcba-dd5d-48b8-9750-d8214ba1383f', 'ip', '-6', 'route', 'list', 'dev', 'tap1fd651e8-79'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 04:08:28.510 6698 DEBUG neutron.agent.linux.utils [req-673313b6-6aed-429e-8a6f-cb98c7430e09 10c05a22cea847f7a42d788fe8f1cd2e 368f193974e14846a186e2cbdb34d110 - - -] Unable to access /var/lib/neutron/external/pids/7251fcba-dd5d-48b8-9750-d8214ba1383f.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-14 04:08:28.512 6698 DEBUG neutron.agent.linux.utils [req-673313b6-6aed-429e-8a6f-cb98c7430e09 10c05a22cea847f7a42d788fe8f1cd2e 368f193974e14846a186e2cbdb34d110 - - -] Unable to access /var/lib/neutron/external/pids/7251fcba-dd5d-48b8-9750-d8214ba1383f.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-14 04:08:28.512 6698 DEBUG neutron.agent.linux.external_process [req-673313b6-6aed-429e-8a6f-cb98c7430e09 10c05a22cea847f7a42d788fe8f1cd2e 368f193974e14846a186e2cbdb34d110 - - -] No process started for 7251fcba-dd5d-48b8-9750-d8214ba1383f disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-14 04:08:28.513 6698 DEBUG oslo_concurrency.lockutils [req-673313b6-6aed-429e-8a6f-cb98c7430e09 10c05a22cea847f7a42d788fe8f1cd2e 368f193974e14846a186e2cbdb34d110 - - -] Releasing semaphore "dhcp-agent-network-lock-7251fcba-dd5d-48b8-9750-d8214ba1383f" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-14 04:08:29.044 6698 DEBUG oslo_concurrency.lockutils [req-673313b6-6aed-429e-8a6f-cb98c7430e09 10c05a22cea847f7a42d788fe8f1cd2e 368f193974e14846a186e2cbdb34d110 - - -] Acquired semaphore "dhcp-agent-network-lock-7251fcba-dd5d-48b8-9750-d8214ba1383f" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-14 04:08:29.045 6698 INFO neutron.agent.dhcp.agent [req-673313b6-6aed-429e-8a6f-cb98c7430e09 10c05a22cea847f7a42d788fe8f1cd2e 368f193974e14846a186e2cbdb34d110 - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=datapath_type=system, ovs_hybrid_plug=True, port_filter=True, binding:vif_type=ovs, binding:vnic_type=normal, created_at=2018-09-14T08:08:25Z, description=, device_id=f9001cef-311c-48d8-8014-c159f6526655, device_owner=compute:nova, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'928c3ff7-8f5a-495c-ac7d-40341d2f7af2', u'ip_address': u'11.197.1.12'}], id=680456d8-5d4f-436f-a825-584188fd39ff, mac_address=fa:16:3e:0d:b7:f1, name=, network_id=7251fcba-dd5d-48b8-9750-d8214ba1383f, port_security_enabled=True, project_id=74e6a538b4564de48b41d0ae5e818844, revision_number=5, security_groups=[u'f45c25cf-f7a2-4b93-8eae-e467548ee8a4'], status=DOWN, tags=[], tenant_id=74e6a538b4564de48b41d0ae5e818844, updated_at=2018-09-14T08:08:28Z
2018-09-14 04:08:29.047 6698 DEBUG neutron.agent.dhcp.agent [req-673313b6-6aed-429e-8a6f-cb98c7430e09 10c05a22cea847f7a42d788fe8f1cd2e 368f193974e14846a186e2cbdb34d110 - - -] Calling driver for network: 7251fcba-dd5d-48b8-9750-d8214ba1383f action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-14 04:08:29.048 6698 DEBUG neutron.agent.linux.dhcp [req-673313b6-6aed-429e-8a6f-cb98c7430e09 10c05a22cea847f7a42d788fe8f1cd2e 368f193974e14846a186e2cbdb34d110 - - -] Building host file: /var/lib/neutron/dhcp/7251fcba-dd5d-48b8-9750-d8214ba1383f/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-14 04:08:29.050 6698 DEBUG neutron.agent.linux.dhcp [req-673313b6-6aed-429e-8a6f-cb98c7430e09 10c05a22cea847f7a42d788fe8f1cd2e 368f193974e14846a186e2cbdb34d110 - - -] Done building host file /var/lib/neutron/dhcp/7251fcba-dd5d-48b8-9750-d8214ba1383f/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-14 04:08:29.053 6698 DEBUG neutron.agent.linux.utils [req-673313b6-6aed-429e-8a6f-cb98c7430e09 10c05a22cea847f7a42d788fe8f1cd2e 368f193974e14846a186e2cbdb34d110 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '15040'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 04:08:29.066 6698 DEBUG neutron.agent.linux.dhcp [req-673313b6-6aed-429e-8a6f-cb98c7430e09 10c05a22cea847f7a42d788fe8f1cd2e 368f193974e14846a186e2cbdb34d110 - - -] Reloading allocations for network: 7251fcba-dd5d-48b8-9750-d8214ba1383f reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-14 04:08:29.069 6698 DEBUG neutron.agent.linux.utils [req-673313b6-6aed-429e-8a6f-cb98c7430e09 10c05a22cea847f7a42d788fe8f1cd2e 368f193974e14846a186e2cbdb34d110 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7251fcba-dd5d-48b8-9750-d8214ba1383f', 'ip', '-4', 'route', 'list', 'dev', 'tap1fd651e8-79'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 04:08:29.082 6698 DEBUG neutron.agent.linux.utils [req-673313b6-6aed-429e-8a6f-cb98c7430e09 10c05a22cea847f7a42d788fe8f1cd2e 368f193974e14846a186e2cbdb34d110 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7251fcba-dd5d-48b8-9750-d8214ba1383f', 'ip', '-6', 'route', 'list', 'dev', 'tap1fd651e8-79'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 04:08:29.097 6698 DEBUG neutron.agent.linux.utils [req-673313b6-6aed-429e-8a6f-cb98c7430e09 10c05a22cea847f7a42d788fe8f1cd2e 368f193974e14846a186e2cbdb34d110 - - -] Unable to access /var/lib/neutron/external/pids/7251fcba-dd5d-48b8-9750-d8214ba1383f.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-14 04:08:29.098 6698 DEBUG neutron.agent.linux.utils [req-673313b6-6aed-429e-8a6f-cb98c7430e09 10c05a22cea847f7a42d788fe8f1cd2e 368f193974e14846a186e2cbdb34d110 - - -] Unable to access /var/lib/neutron/external/pids/7251fcba-dd5d-48b8-9750-d8214ba1383f.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-14 04:08:29.098 6698 DEBUG neutron.agent.linux.external_process [req-673313b6-6aed-429e-8a6f-cb98c7430e09 10c05a22cea847f7a42d788fe8f1cd2e 368f193974e14846a186e2cbdb34d110 - - -] No process started for 7251fcba-dd5d-48b8-9750-d8214ba1383f disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-14 04:08:29.099 6698 DEBUG oslo_concurrency.lockutils [req-673313b6-6aed-429e-8a6f-cb98c7430e09 10c05a22cea847f7a42d788fe8f1cd2e 368f193974e14846a186e2cbdb34d110 - - -] Releasing semaphore "dhcp-agent-network-lock-7251fcba-dd5d-48b8-9750-d8214ba1383f" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-14 04:08:52.142 6698 DEBUG oslo_concurrency.lockutils [req-3fc0d939-ef6f-4826-a6e5-565c76ecfe05 d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Acquired semaphore "dhcp-agent-network-lock-990b4ddf-dd04-44e0-920c-02fce451b987" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-14 04:08:52.143 6698 DEBUG oslo_concurrency.lockutils [req-3fc0d939-ef6f-4826-a6e5-565c76ecfe05 d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Releasing semaphore "dhcp-agent-network-lock-990b4ddf-dd04-44e0-920c-02fce451b987" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-14 04:08:53.177 6698 DEBUG oslo_concurrency.lockutils [req-3fc0d939-ef6f-4826-a6e5-565c76ecfe05 d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Acquired semaphore "dhcp-agent-network-lock-990b4ddf-dd04-44e0-920c-02fce451b987" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-14 04:08:53.177 6698 DEBUG oslo_concurrency.lockutils [req-3fc0d939-ef6f-4826-a6e5-565c76ecfe05 d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Releasing semaphore "dhcp-agent-network-lock-990b4ddf-dd04-44e0-920c-02fce451b987" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-14 04:08:53.950 6698 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-14 04:08:53.953 6698 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-14 04:09:53.950 6698 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-14 04:09:53.954 6698 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.004s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-14 04:10:53.951 6698 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-14 04:10:53.954 6698 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.004s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-14 04:11:53.951 6698 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-14 04:11:53.954 6698 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.004s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-14 04:12:40.407 6698 DEBUG oslo_concurrency.lockutils [req-d0a05a6c-3897-4cbf-b534-601cc9ff400b d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Acquired semaphore "dhcp-agent-network-lock-7251fcba-dd5d-48b8-9750-d8214ba1383f" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-14 04:12:40.408 6698 DEBUG neutron.agent.dhcp.agent [req-d0a05a6c-3897-4cbf-b534-601cc9ff400b d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Calling driver for network: 7251fcba-dd5d-48b8-9750-d8214ba1383f action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-14 04:12:40.411 6698 DEBUG neutron.agent.linux.utils [req-d0a05a6c-3897-4cbf-b534-601cc9ff400b d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7251fcba-dd5d-48b8-9750-d8214ba1383f', 'dhcp_release', 'tap1fd651e8-79', '11.197.1.12', 'fa:16:3e:0d:b7:f1'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 04:12:40.433 6698 DEBUG neutron.agent.linux.dhcp [req-d0a05a6c-3897-4cbf-b534-601cc9ff400b d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Building host file: /var/lib/neutron/dhcp/7251fcba-dd5d-48b8-9750-d8214ba1383f/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-14 04:12:40.437 6698 DEBUG neutron.agent.linux.dhcp [req-d0a05a6c-3897-4cbf-b534-601cc9ff400b d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Done building host file /var/lib/neutron/dhcp/7251fcba-dd5d-48b8-9750-d8214ba1383f/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-14 04:12:40.445 6698 DEBUG neutron.agent.linux.utils [req-d0a05a6c-3897-4cbf-b534-601cc9ff400b d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '15040'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 04:12:40.457 6698 DEBUG neutron.agent.linux.dhcp [req-d0a05a6c-3897-4cbf-b534-601cc9ff400b d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Reloading allocations for network: 7251fcba-dd5d-48b8-9750-d8214ba1383f reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-14 04:12:40.458 6698 DEBUG neutron.agent.linux.utils [req-d0a05a6c-3897-4cbf-b534-601cc9ff400b d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7251fcba-dd5d-48b8-9750-d8214ba1383f', 'ip', '-4', 'route', 'list', 'dev', 'tap1fd651e8-79'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 04:12:40.474 6698 DEBUG neutron.agent.linux.utils [req-d0a05a6c-3897-4cbf-b534-601cc9ff400b d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7251fcba-dd5d-48b8-9750-d8214ba1383f', 'ip', '-6', 'route', 'list', 'dev', 'tap1fd651e8-79'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 04:12:40.490 6698 DEBUG neutron.agent.linux.utils [req-d0a05a6c-3897-4cbf-b534-601cc9ff400b d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Unable to access /var/lib/neutron/external/pids/7251fcba-dd5d-48b8-9750-d8214ba1383f.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-14 04:12:40.491 6698 DEBUG neutron.agent.linux.utils [req-d0a05a6c-3897-4cbf-b534-601cc9ff400b d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Unable to access /var/lib/neutron/external/pids/7251fcba-dd5d-48b8-9750-d8214ba1383f.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-14 04:12:40.491 6698 DEBUG neutron.agent.linux.external_process [req-d0a05a6c-3897-4cbf-b534-601cc9ff400b d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] No process started for 7251fcba-dd5d-48b8-9750-d8214ba1383f disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-14 04:12:40.492 6698 DEBUG oslo_concurrency.lockutils [req-d0a05a6c-3897-4cbf-b534-601cc9ff400b d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Releasing semaphore "dhcp-agent-network-lock-7251fcba-dd5d-48b8-9750-d8214ba1383f" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-14 04:12:53.951 6698 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-14 04:12:53.956 6698 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.004s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-14 04:13:26.543 6698 DEBUG oslo_concurrency.lockutils [req-3d35dc9b-7c5d-44bd-935e-cae5e74eb19f d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Acquired semaphore "dhcp-agent-network-lock-7251fcba-dd5d-48b8-9750-d8214ba1383f" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-14 04:13:26.545 6698 DEBUG neutron.agent.dhcp.agent [req-3d35dc9b-7c5d-44bd-935e-cae5e74eb19f d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Calling driver for network: 7251fcba-dd5d-48b8-9750-d8214ba1383f action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-14 04:13:26.546 6698 DEBUG neutron.agent.linux.utils [req-3d35dc9b-7c5d-44bd-935e-cae5e74eb19f d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7251fcba-dd5d-48b8-9750-d8214ba1383f', 'dhcp_release', 'tap1fd651e8-79', '11.197.1.1', 'fa:16:3e:f8:ff:59'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 04:13:26.564 6698 DEBUG neutron.agent.linux.dhcp [req-3d35dc9b-7c5d-44bd-935e-cae5e74eb19f d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Building host file: /var/lib/neutron/dhcp/7251fcba-dd5d-48b8-9750-d8214ba1383f/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-14 04:13:26.570 6698 DEBUG neutron.agent.linux.dhcp [req-3d35dc9b-7c5d-44bd-935e-cae5e74eb19f d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Done building host file /var/lib/neutron/dhcp/7251fcba-dd5d-48b8-9750-d8214ba1383f/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-14 04:13:26.576 6698 DEBUG neutron.agent.linux.utils [req-3d35dc9b-7c5d-44bd-935e-cae5e74eb19f d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '15040'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 04:13:26.586 6698 DEBUG neutron.agent.linux.dhcp [req-3d35dc9b-7c5d-44bd-935e-cae5e74eb19f d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Reloading allocations for network: 7251fcba-dd5d-48b8-9750-d8214ba1383f reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-14 04:13:26.588 6698 DEBUG neutron.agent.linux.utils [req-3d35dc9b-7c5d-44bd-935e-cae5e74eb19f d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7251fcba-dd5d-48b8-9750-d8214ba1383f', 'ip', '-4', 'route', 'list', 'dev', 'tap1fd651e8-79'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 04:13:26.602 6698 DEBUG neutron.agent.linux.utils [req-3d35dc9b-7c5d-44bd-935e-cae5e74eb19f d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7251fcba-dd5d-48b8-9750-d8214ba1383f', 'ip', '-6', 'route', 'list', 'dev', 'tap1fd651e8-79'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 04:13:26.620 6698 DEBUG neutron.agent.linux.utils [req-3d35dc9b-7c5d-44bd-935e-cae5e74eb19f d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Unable to access /var/lib/neutron/external/pids/7251fcba-dd5d-48b8-9750-d8214ba1383f.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-14 04:13:26.621 6698 DEBUG neutron.agent.linux.utils [req-3d35dc9b-7c5d-44bd-935e-cae5e74eb19f d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Unable to access /var/lib/neutron/external/pids/7251fcba-dd5d-48b8-9750-d8214ba1383f.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-14 04:13:26.622 6698 DEBUG neutron.agent.linux.external_process [req-3d35dc9b-7c5d-44bd-935e-cae5e74eb19f d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] No process started for 7251fcba-dd5d-48b8-9750-d8214ba1383f disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-14 04:13:26.623 6698 DEBUG oslo_concurrency.lockutils [req-3d35dc9b-7c5d-44bd-935e-cae5e74eb19f d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Releasing semaphore "dhcp-agent-network-lock-7251fcba-dd5d-48b8-9750-d8214ba1383f" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-14 04:13:53.952 6698 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-14 04:13:53.955 6698 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-14 04:13:55.066 6698 DEBUG oslo_concurrency.lockutils [req-3553aee4-8de7-4228-bbce-0d7822565bdf d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Acquired semaphore "dhcp-agent-network-lock-7251fcba-dd5d-48b8-9750-d8214ba1383f" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-14 04:13:55.067 6698 DEBUG neutron.agent.dhcp.agent [req-3553aee4-8de7-4228-bbce-0d7822565bdf d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Calling driver for network: 7251fcba-dd5d-48b8-9750-d8214ba1383f action: disable call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-14 04:13:55.071 6698 DEBUG neutron.agent.linux.utils [req-3553aee4-8de7-4228-bbce-0d7822565bdf d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Running command (rootwrap daemon): ['kill', '-9', '15040'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 04:13:55.087 6698 DEBUG neutron.agent.linux.utils [req-3553aee4-8de7-4228-bbce-0d7822565bdf d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Running command: ['ip', '-o', 'link', 'show', 'br-int'] create_process /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:92
2018-09-14 04:13:55.113 6698 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=0): DelPortCommand(if_exists=True, bridge=br-int, port=tap1fd651e8-79) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-09-14 04:13:55.190 6698 DEBUG neutron.agent.linux.utils [req-7940a28b-108f-4c14-b327-2df6666f5185 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 04:13:55.203 6698 DEBUG neutron.agent.linux.utils [req-7940a28b-108f-4c14-b327-2df6666f5185 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'delete', 'qdhcp-7251fcba-dd5d-48b8-9750-d8214ba1383f'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 04:13:55.216 6698 DEBUG oslo_concurrency.lockutils [req-7940a28b-108f-4c14-b327-2df6666f5185 - - - - -] Releasing semaphore "dhcp-agent-network-lock-7251fcba-dd5d-48b8-9750-d8214ba1383f" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-14 04:14:04.379 6698 DEBUG oslo_concurrency.lockutils [req-905cb726-a4a5-485a-9bd9-e2c8eb766e2a d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Acquired semaphore "dhcp-agent-network-lock-7251fcba-dd5d-48b8-9750-d8214ba1383f" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-14 04:14:04.737 6698 DEBUG neutron.agent.linux.utils [req-a3a125cf-afcf-4f25-b7ca-c76332c7a54d - - - - -] Unable to access /var/lib/neutron/external/pids/7251fcba-dd5d-48b8-9750-d8214ba1383f.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-14 04:14:04.738 6698 DEBUG neutron.agent.linux.utils [req-a3a125cf-afcf-4f25-b7ca-c76332c7a54d - - - - -] Unable to access /var/lib/neutron/external/pids/7251fcba-dd5d-48b8-9750-d8214ba1383f.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-14 04:14:04.738 6698 DEBUG neutron.agent.linux.external_process [req-a3a125cf-afcf-4f25-b7ca-c76332c7a54d - - - - -] No process started for 7251fcba-dd5d-48b8-9750-d8214ba1383f disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-14 04:14:04.740 6698 DEBUG neutron.agent.dhcp.agent [req-a3a125cf-afcf-4f25-b7ca-c76332c7a54d - - - - -] Calling driver for network: 7251fcba-dd5d-48b8-9750-d8214ba1383f action: disable call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-14 04:14:04.742 6698 DEBUG neutron.agent.linux.utils [req-a3a125cf-afcf-4f25-b7ca-c76332c7a54d - - - - -] Unable to access /var/lib/neutron/dhcp/7251fcba-dd5d-48b8-9750-d8214ba1383f/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-14 04:14:04.744 6698 DEBUG neutron.agent.linux.utils [req-a3a125cf-afcf-4f25-b7ca-c76332c7a54d - - - - -] Unable to access /var/lib/neutron/dhcp/7251fcba-dd5d-48b8-9750-d8214ba1383f/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-14 04:14:04.746 6698 DEBUG neutron.agent.linux.external_process [req-a3a125cf-afcf-4f25-b7ca-c76332c7a54d - - - - -] No process started for 7251fcba-dd5d-48b8-9750-d8214ba1383f disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-14 04:14:04.748 6698 DEBUG neutron.agent.linux.dhcp [req-a3a125cf-afcf-4f25-b7ca-c76332c7a54d - - - - -] Unable to access /var/lib/neutron/dhcp/7251fcba-dd5d-48b8-9750-d8214ba1383f/interface _get_value_from_conf_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:271
2018-09-14 04:14:04.748 6698 DEBUG neutron.agent.linux.dhcp [req-a3a125cf-afcf-4f25-b7ca-c76332c7a54d - - - - -] No interface exists for network 7251fcba-dd5d-48b8-9750-d8214ba1383f destroy /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1487
2018-09-14 04:14:04.864 6698 DEBUG neutron.agent.linux.utils [req-9a8848d8-1fb3-4acb-a752-a51cf388d8f3 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-14 04:14:04.892 6698 DEBUG neutron.agent.linux.dhcp [req-9a8848d8-1fb3-4acb-a752-a51cf388d8f3 - - - - -] Namespace already deleted: qdhcp-7251fcba-dd5d-48b8-9750-d8214ba1383f _destroy_namespace_and_port /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:249
2018-09-14 04:14:04.918 6698 DEBUG oslo_concurrency.lockutils [req-9a8848d8-1fb3-4acb-a752-a51cf388d8f3 - - - - -] Releasing semaphore "dhcp-agent-network-lock-7251fcba-dd5d-48b8-9750-d8214ba1383f" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-14 04:14:06.048 6698 DEBUG oslo_concurrency.lockutils [req-905cb726-a4a5-485a-9bd9-e2c8eb766e2a d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Acquired semaphore "dhcp-agent-network-lock-7251fcba-dd5d-48b8-9750-d8214ba1383f" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-14 04:14:06.050 6698 DEBUG oslo_concurrency.lockutils [req-905cb726-a4a5-485a-9bd9-e2c8eb766e2a d5ce6e22423945b3856059aa9d9f9c35 24a4048b7dbb4950873e581bf0067256 - - -] Releasing semaphore "dhcp-agent-network-lock-7251fcba-dd5d-48b8-9750-d8214ba1383f" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-14 04:14:07.469 6698 DEBUG neutron.agent.dhcp.agent [req-4197fad4-b267-4cf2-89bc-4b9638f2b3fd - - - - -] resync (7251fcba-dd5d-48b8-9750-d8214ba1383f): ['Agent port was deleted'] _periodic_resync_helper /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:256
2018-09-14 04:14:07.470 6698 INFO neutron.agent.dhcp.agent [req-4197fad4-b267-4cf2-89bc-4b9638f2b3fd - - - - -] Synchronizing state
2018-09-14 04:14:08.537 6698 INFO neutron.agent.dhcp.agent [req-a8952051-c930-4386-abd4-107d20dbe729 - - - - -] All active networks have been fetched through RPC.
2018-09-14 04:14:08.539 6698 INFO neutron.agent.dhcp.agent [-] Starting network 990b4ddf-dd04-44e0-920c-02fce451b987 dhcp configuration
2018-09-14 04:14:08.539 6698 INFO neutron.agent.dhcp.agent [-] Finished network 990b4ddf-dd04-44e0-920c-02fce451b987 dhcp configuration
2018-09-14 04:14:08.540 6698 INFO neutron.agent.dhcp.agent [req-a8952051-c930-4386-abd4-107d20dbe729 - - - - -] Synchronizing state complete
